test/parallel/test-net-stream.js

```diff
@@ -69,5 +69,7 @@ const server = net.createServer(function(socket) {
 });
 
 process.on('exit', function() {
-  assert.strictEqual(server.connections, 0);
+  server.getConnections((err, connections) => {
```

Comment:
You can't do asynchronous work in a `process` `'exit'` handler.

Reply:
Thanks Trott! I'll delete this one on the next revision